packages/payment-proxy-client/src/constants.ts

```diff
@@ -15,9 +13,23 @@ export const initialChannelBalance = parseInt(
   import.meta.env.VITE_INITIAL_CHANNEL_BALANCE,
   10
 );
+const ENV_VAR_SPLIT_CHAR = ";";
```

Comment:
I switched to `;` as the delimiter as some of the image names in #1753  contain a `,`